homeassistant/helpers/template.py

```diff
@@ -168,10 +168,10 @@ class ResultWrapper:
     render_result: str | None
 
 
-def gen_result_wrapper(kls):
+def gen_result_wrapper(kls: type[dict | list | set]) -> type:
     """Generate a result wrapper."""
 
-    class Wrapper(kls, ResultWrapper):
+    class Wrapper(kls, ResultWrapper):  # type: ignore[valid-type,misc]
```

Comment:
Why is this ignore needed?

Reply:
Mypy doesn't recognize dynamic base classes